meta.yaml

```diff
@@ -32,9 +32,26 @@ requirements:
     - tqdm
     # for run_exports
     - {{ compiler('cxx') }}
+  run_constrained:
+    - _cuda >=11.0
```

Comment:
```suggestion
    - __cuda >=11.0
```

Reply:
This breaks our builds `__cuda` as shown above

Reply:
`_cuda` doesn't do anything AFAIK. It has to have 2 `_`s.

Reply:
If we can't build with this inside of a `nvidia/cuda` container then this is a large problem

Reply:
We can disable by setting `CONDA_OVERRIDE_CUDA=None`.

Reply:
If needed you can override the detection `__cuda` version by setting the  `CONDA_OVERRIDE_CUDA` environment variable.

Reply:
Thanks for catching this @jakirkham.  I suspect some of the odd behavior I was see is due to me using a single `_` cuda.